### PR TITLE
Search feature for owners

### PR DESCRIPTION
**Pull request by simrin051**

### CODE REVIEW

**Comment by simrin051** [comment]
Hi @arey 

Can you please merge if everything is ok with code

File: src/app/owners/owner-list/owner-list.component.ts [hunk 1]
```diff
@@ -51,4 +53,25 @@ export class OwnerListComponent implements OnInit {
...
 
+  searchByLastName(lastName: string)
+  {
+      console.log("inside search by last name starting with "+(lastName));
+      this.ownerService.searchOwners(lastName)
+      .subscribe(
+      (owners)=>{
+
+       this.owners = owners;
+       document.getElementById("ownersTable").style.visibility="visible";
```
Comment (arey) [inline src/app/owners/owner-list/owner-list.component.ts:64]: Not sure that accessing to the DOM for setting CSS is the best way

Comment (simrin051) [reply src/app/owners/owner-list/owner-list.component.ts:64]: Can you please  advice  any other way.

Comment (arey) [reply src/app/owners/owner-list/owner-list.component.ts:64]: Maybe look at the `[hidden]` attribute or the `*ngIf`directive?

**Comment by simrin051** [comment]
@arey 

Can you please review the code.

**Comment by simrin051** [comment]
@arey

Can you please review the code.

**Comment by simrin051** [comment]
@arey 

Can you please review the code.

**Comment by arey** [comment]
Good job @simrin051. Thanks

**Pull request merged**